v5_enum.go

```diff
@@ -317,13 +317,31 @@ const (
 	WithdrawStatusV5BlockchainConfirmed = WithdrawStatusV5("BlockchainConfirmed")
 )
 
+// IsLowestRisk :
 type IsLowestRisk int
 
 const (
-	IsLowestRiskFalse = IsLowestRisk(0)
-	IsLowestRiskTrue  = IsLowestRisk(1)
+	// IsLowestRiskNo :
+	IsLowestRiskNo = IsLowestRisk(0)
+	// IsLowestRiskYes :
+	IsLowestRiskYes = IsLowestRisk(1)
```

Comment:
This is a change that has nothing to do with the purpose of PR.

There's a possibility that unnecessary actions might be requested from the user, so adjustments may be made later.